.travis.yml

```diff
@@ -19,11 +19,13 @@ install:
   - tar -xvf $PWD/travis-phantomjs/phantomjs-2.0.0-ubuntu-12.04.tar.bz2 -C $PWD/travis-phantomjs
   - export PATH=$PWD/travis-phantomjs:$PATH
 
+  - cp ./api/.travis.env ./api/.env
+
   - travis_retry npm install --no-spin --quiet -g gulp http-server
   - travis_retry npm install --no-spin
   - travis_retry gulp bower:install
   - travis_retry composer self-update
-  - travis_retry composer install --prefer-source --no-interaction --working-dir api
+  - travis_retry composer install --prefer-source --no-interaction --optimize-autoloader --working-dir api
```

Comment:
@artstorm ah okay that makes sense now, ok can you please make that change with this PR, it will probably be a good idea to have it turned off so that the build fails when we have a non-conformant to PSR file name.